### PR TITLE
Updated event names and fixed log event.

### PR DESCRIPTION
**Pull request by jonathanswong**
- updated event names 
- updated standard events to first grab an event parameters payload before logging the event

### CODE REVIEW

File: facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt [hunk 1]
```diff
@@ -21,151 +21,147 @@ object Commands {
...
-    const val EVENT_NAME_LIVE_STREAMING_START = AppEventsConstants.EVENT_NAME_LIVE_STREAMING_START
-    const val EVENT_NAME_LIVE_STREAMING_STOP = AppEventsConstants.EVENT_NAME_LIVE_STREAMING_STOP
-    const val EVENT_NAME_LIVE_STREAMING_PAUSE = AppEventsConstants.EVENT_NAME_LIVE_STREAMING_PAUSE
-    const val EVENT_NAME_LIVE_STREAMING_RESUME = AppEventsConstants.EVENT_NAME_LIVE_STREAMING_RESUME
-    const val EVENT_NAME_LIVE_STREAMING_ERROR = AppEventsConstants.EVENT_NAME_LIVE_STREAMING_ERROR
-    const val EVENT_NAME_LIVE_STREAMING_UPDATE_STATUS =
-        AppEventsConstants.EVENT_NAME_LIVE_STREAMING_UPDATE_STATUS
-    const val EVENT_NAME_PRODUCT_CATALOG_UPDATE =
-        AppEventsConstants.EVENT_NAME_PRODUCT_CATALOG_UPDATE
+    const val ACTIVATED_APP = "activiatedapp"
```
Comment (jameskeith) [inline facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt]: `"activiatedapp"` - typo?

File: facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt [hunk 2]
```diff
@@ -21,151 +21,147 @@ object Commands {
...
-        AppEventsConstants.EVENT_NAME_AD_IMPRESSION,
-        AppEventsConstants.EVENT_NAME_AD_CLICK,
-        AppEventsConstants.EVENT_NAME_LIVE_STREAMING_START,
-        AppEventsConstants.EVENT_NAME_LIVE_STREAMING_STOP,
-        AppEventsConstants.EVENT_NAME_LIVE_STREAMING_PAUSE,
-        AppEventsConstants.EVENT_NAME_LIVE_STREAMING_RESUME,
-        AppEventsConstants.EVENT_NAME_LIVE_STREAMING_ERROR,
-        AppEventsConstants.EVENT_NAME_LIVE_STREAMING_UPDATE_STATUS,
-        AppEventsConstants.EVENT_NAME_PRODUCT_CATALOG_UPDATE
+        "activiatedapp",
```
Comment (jameskeith) [inline facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt]: `"activiatedapp"` - typo?

File: facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt [hunk 3]
```diff
@@ -21,151 +21,147 @@ object Commands {
...
+    const val LIVE_STREAMING_STOP = "livestreamingstop"
+    const val LIVE_STREAMING_PAUSE = "livestreamingpause"
+    const val LIVE_STREAMING_RESUME = "livestreamingresume"
+    const val LIVE_STREAMING_ERROR = "livestreamingerror"
+    const val LIVE_STREAMING_UPDATE_STATUS = "livestreamingupdatestatus"
+    const val PRODUCT_CATALOG_UPDATE = "productcatalogupdate"
 }
 
 object StandardEvents {
     val standardEventNames = arrayOf(
```
Comment (jameskeith) [inline facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt]: I understand switching to our own event naming, but these look like duplicates of the strings above. Can we not reference those in this array so we don't have the same strings twice?

File: facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt [hunk 4]
```diff
@@ -21,151 +21,147 @@ object Commands {
...
+    const val DESCRIPTION = AppEventsConstants.EVENT_PARAM_DESCRIPTION
+    const val SOURCE_APPLICATION = AppEventsConstants.EVENT_PARAM_SOURCE_APPLICATION
+    const val VALUE_YES = AppEventsConstants.EVENT_PARAM_VALUE_YES
+    const val VALUE_NO = AppEventsConstants.EVENT_PARAM_VALUE_NO
+    const val AD_TYPE = AppEventsConstants.EVENT_PARAM_AD_TYPE
+    const val ORDER_ID = AppEventsConstants.EVENT_PARAM_ORDER_ID
+    const val PRODUCT_CUSTOM_LABEL_0 =
         AppEventsConstants.EVENT_PARAM_PRODUCT_CUSTOM_LABEL_0
-    const val EVENT_PARAM_PRODUCT_CUSTOM_LABEL_1 =
+    const val PRODUCT_CUSTOM_LABEL_1 =
```
Comment (jameskeith) [inline facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt]: All the other `PRODUCT_CUSTOM_LABEL_#` are prefixed by `PARAM_`
Suggest we bring Label 0 and 1 in-line with the others.

Comment (christinasund) [reply facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt]: The actual string value for the constant is "fb_product_custom_label_0", so I think that is the reason for the change here.

Comment (jameskeith) [reply facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt]: sure - it's not so obvious on the PR page, but the rest of the product custom labels are named differently:
const val PRODUCT_CUSTOM_LABEL_1 = ...
vs
const val PARAM_PRODUCT_CUSTOM_LABEL_3 = ...

Comment (christinasund) [reply facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt]: Oh got it, I see what you mean ... yeah agree on consistency then

Comment (jonathanswong) [reply facebook/src/main/java/com/tealium/remotecommands/facebook/FacebookConstants.kt]: Must have missed it with multi-cursor select.

**Pull request merged**